src/client/hooks/analytics.hook.ts

```diff
@@ -149,3 +149,26 @@ export function trackItemClick(
 ) {
   return trackLink(url, linkTitle, [profileTypeDimension(profileType)]);
 }
+
+export function trackDownload(
+  downloadKind: string,
+  documentKind: string,
```

Comment:
Wat is een documentKind? Kun je hier specifiekere types neerzetten? Is het niet gewoon `fileType`? 

Buiten dat, volgens mij mogen we alleen maar PDF aanbieden als download.

Reply:
Bij bezwaren gaan we gebruikers ook de mogelijkheid bieden zelf geuploade bestanden opnieuw te downloaden. Dat kunnen allerlei bestanden zijn.